### ISSUE TITLE
Sync doesn't work with VS Code for the web

### ISSUE THREAD

**Issue by Kazuma-chan** [microsoft/vscode#135574]
When I try to sync, it waits a moment and fails. Log doesn't show anything (It shows stuff when I reload, but if I clear it, it just keeps empty, even when I try to sync). I'm using Vivaldi browser, but I tried on chromium and got the same result. Using github account.

Version: 1.61.2
Commit: 6cba118ac49a1b88332f312a8f67186f7f3c1643
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.81 Safari/537.36
Embedder: vscode.dev

**Comment by sandy081** [comment]
Can you please record a video/gif and share with us to know where it is failing.

**Comment by Kazuma-chan** [comment]
Sorry for the late reply, I just tried doing that both in the vscode.dev and in the vscode in my pc, but idk what I'm looking for since I don't know much about dev tools. I did manage to record [something](https://imgur.com/a/Zqa18my) though using the network tab, idk if this helps, but it appeared as soon as I clicked the sync button. I also downloaded a ".har" file, tell me if I should send it too.
